_redesign_use_cases/search-vector-database.markdown

```diff
@@ -5,7 +5,7 @@ description: 'OpenSearch as a vector database supports a range of applications.
 feature_area_category_name: Search
 feature_area_solution_name: Vector Database
 how_to_get_started: 
-  - 'You can begin exploring OpenSearch&apos;s vector database functionality by downloading your preferred distribution. To learn more or start a discussion, join the Slack channel or check out our user forum and follow our blog for the latest on OpenSearch tools.'
+  - 'You can begin exploring OpenSearch&apos;s vector database functionality by exploring <a href="https://opensearch.org/docs/latest/search-plugins/vector-search/" target="_blank">Vector search</a> in our documentation. To learn more or start a discussion, join the Slack channel or check out our user forum and follow our blog for the latest on OpenSearch tools.'
```

Comment:
The first sentence would read better as "by exploring our vector search documentation." Also, let's add links to the Slack channel, the user forum, and the blog. Thanks!